src/pages/Reviews.js

```diff
@@ -63,21 +62,24 @@ const Reviews = ({ productId }) => {
         rating: '',
         title: '',
         comment: '',
-        product: productId,
-        user: user,
```

Comment:
Same reason as above, we don't need to pass user in the request body: https://github.com/NatachaKey/reactEcommerce/pull/14/files#r1246578739

Reply:
@akosasante is  product: productId, necessary here? i mean i see that you've deleted it, i merged your branch to mine order, than i merged mine to main and did git pull origin main in main locally but i still have this line

Reply:
I mean since Review model is looking for the product being passed from frontend it should be there, right?
![image](https://github.com/NatachaKey/reactEcommerce/assets/103755090/07cc54ca-ea66-45ff-97b3-c6954e4f05a0)